src/mountscript.sh

```diff
@@ -186,6 +186,14 @@ check_turbo_option()
     if [[ "$MOUNT_OPTIONS" =~ $matchstr ]]; then
         export USING_AZNFSCLIENT=true
     fi
+    
+    #
+    # Check if fuse flag is passed. We need to support both options for some reason.
```

Comment:
We need to support both options as some customer using turbo option.
TODO: Once we have confirmation no customer using should remove turbo option.